src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurityNative
+    {
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_ReleaseGssBuffer")]
+        internal static extern void ReleaseGssBuffer(
+            IntPtr bufferPtr,
+            UInt64 length);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_DisplayStatus")]
+        internal static extern Status DisplayStatus(
+            out Status minorStatus,
+            Status statusValue,
+            bool isGssMechCode,
+            ref GssBuffer buffer);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_ImportUserName")]
+        internal static extern Status ImportUserName(
+            out Status minorStatus,
+            string inputName,
+            int inputNameLen,
```

Comment:
Here in managed code, a string followed by a Length value suggests that str.Length is the value to pass (`cch` in Win32 Hungarian-prefix); but by usage this looks like it wants the length of the encoded string, which is UTF-8 in this case (making it `cb`).

It didn't take very long to verify, but I wonder if the parameter names either using the Hungarian or be named something like `inputNameByteCount` would help eliminate future bugs.